### ISSUE TITLE
Nails & Rivets, both destructible and indestructible.

### ISSUE THREAD

**Issue by Err03r** [studio-minus/ppg-bugs-and-requests#3471]
### Feature description

Fasteners that attach two objects together, and restrict movement and rotation between them, but not the map. They work just like they do in real life.

### Why should you spend time implementing this?

1. Nails & Rivets may provide a potentially less buggier way to attach objects together. Everyone who has experience building contraptions knows how buggy the cables can be. Think of them as the pin, that is already in the game, except, it doesn't allow for any rotation, or movement.

2. Nails & Rivets provide a neater, and cleaner way to build contraptions. Mods won't be needed to hide cables because of how ugly they look on contraptions.

### What made me request them?

1. The buggy nature of the cables.
2. Wanting a way to rigidly attach objects together, to the point where there is no movement at all.
3. A way to make contraptions without having to stare at messy, ugly cables.

### What is its purpose?

To provide a way to attach two objects together, and restrict movement and rotation between them, but not the map.

### Am I sure it doesn't already exist in some form?

A way to attach two objects together, and restrict movement and rotation between them already exists, but can be buggy, and suffers from limitations, like bending when a heavy mass is imposed on it, when it's supposed to be rigid. That is why I am proposing Nails & Rivets. It would be just like the pin, that is already in the game, except, it doesn't allow for any rotation, or movement.

### Additional context

Please experiment with this idea, Mestiez.

**Comment by mestiez** [comment]
Technically these would rely on the same mechanism as fixed cables, so this probably won't provide any more stability.

**Comment by LexHaran** [comment]
good idea but im pretty sure you can just use edit layer to hide fixed cables or use steel binding that is usually hidden in the back layer when spawned (might only be for me)
